qt/aqt/forms/fields.ui

```diff
@@ -104,31 +108,47 @@
        </property>
       </widget>
      </item>
-     <item row="1" column="0">
-      <widget class="QLabel" name="label_18">
+     <item row="2" column="0">
+      <widget class="QLabel" name="label_sort">
        <property name="text">
         <string>actions_options</string>
        </property>
       </widget>
      </item>
-     <item row="1" column="1">
+     <item row="2" column="1">
       <widget class="QRadioButton" name="sortField">
        <property name="text">
         <string>fields_sort_by_this_field_in_the</string>
        </property>
       </widget>
      </item>
-     <item row="2" column="1">
-      <widget class="QCheckBox" name="rtl">
+     <item row="0" column="0">
+      <widget class="QLabel" name="label_desc">
+       <property name="sizePolicy">
+        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+         <horstretch>0</horstretch>
+         <verstretch>0</verstretch>
+        </sizepolicy>
+       </property>
        <property name="text">
-        <string>fields_reverse_text_direction_rtl</string>
+        <string>fields_description</string>
```

Comment:
In Designer, please go to Edit>Edit Tab Order, and ensure the newly added field comes in the correct order. You can right click + use "start from here"